### ISSUE TITLE
Assign SSH KEY for use API Django

### ISSUE THREAD

**Issue by cgriva** [nuxsmin/sysPass#197]
Its a possible to use SSH KEY with framework Django ?

**Comment by nuxsmin** [comment]
Did you mean to connect django auth with sysPass?

**Comment by cgriva** [comment]
yes I will wish to use the API DJANGO AUTH to perform an update environments

**Comment by cgriva** [comment]
Thanks !

**Comment by vmario89** [comment]
i think some collection of requested auth backends for sysPass should be done. I stumpled upon:

- different LDAP implementations
- SSO
- OAuth
- Kerberos
- Google API
- Django

I guess there are tons of other issues asking similar stuff
